js/script.js

```diff
@@ -535,19 +571,75 @@ function ot_buildMatcher() {
 
 var ot_validCombos = {
 
-    "ot_linux,ot_ORTModule,ot_X64,ot_CUDA":
+    "ot_linux,ot_large_model,ot_python,ot_X64,ot_CUDA":
     "pip install onnxruntime-training -f https://downloads.onnxruntime.ai/onnxruntime_stable_<b>&lt;cu_version*</b>&gt;.html<br/>pip install torch-ort<br/>python -m torch_ort.configure<br/><br/>*</b><a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
 
-    "ot_linux,ot_ORTModule,ot_X64,ot_ROCm":
-    "pip install onnxruntime-training -f https://downloads.onnxruntime.ai/onnxruntime_stable_<b>&lt;rocm_version*</b>&gt;.html<br/>pip install torch-ort<br/>python -m torch_ort.configure<br/><br/>*<a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>"
+    "ot_linux,ot_large_model,ot_python,ot_X64,ot_ROCm":
+    "pip install onnxruntime-training -f https://downloads.onnxruntime.ai/onnxruntime_stable_<b>&lt;rocm_version*</b>&gt;.html<br/>pip install torch-ort<br/>python -m torch_ort.configure<br/><br/>*<a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
+    
+    "ot_linux,ot_on_device,ot_python,ot_X64,ot_CPU":
+    "pip install onnxruntime-training -f https://downloads.onnxruntime.ai/onnxruntime_stable_<b>&lt;cu_version*</b>&gt;.html<br/><br/>*</b><a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
+
+    "ot_linux,ot_on_device,ot_python,ot_X64,ot_CUDA":
+    "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
+
+    "ot_windows,ot_on_device,ot_python,ot_X64,ot_CPU":
+    "pip install onnxruntime-training -f https://downloads.onnxruntime.ai/onnxruntime_stable_<b>&lt;cu_version*</b>&gt;.html<br/><br/>*</b><a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
```

Comment:
Windows installation for python is not from ```https://downloads.onnxruntime.ai/onnxruntime_stable_```.

Can you point to building from source for now? We need to create a python package for windows cpu.